src/inspector/internal/services/elementrepositoryservice.cpp

```diff
@@ -183,6 +175,18 @@ QList<mu::engraving::EngravingItem*> ElementRepositoryService::findChords() cons
             continue;
         }
 
+        if (element->type() == mu::engraving::ElementType::BEAM) {
+            const mu::engraving::Beam* beam = mu::engraving::toBeam(element);
+
+            for (mu::engraving::ChordRest* chordRest : beam->elements()) {
+                if (!chordRest->isChord()) {
+                    continue;
+                }
+                elements << mu::engraving::toChord(chordRest);
```

Comment:
The same applies here:
```suggestion
                elements << chordRest;
```

Reply:
Thanks! Fixed